data/xml/settings-public.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<settings xmlns="http://maven.apache.org/SETTINGS/1.2.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.2.0 http://maven.apache.org/xsd/settings-1.2.0.xsd">
+  <pluginGroups>
+  </pluginGroups>
+
+  <proxies>
+  </proxies>
+
+  <servers>
+    <server>
+       <id>tackle-testapp-public</id>
```

Comment:
If the only difference in setting.xml and this xml is id , username and password then we can modify these 3 using the writeMavenSettingsFile method from utils.
Please check once.